website/docusaurus.config.js

```diff
@@ -58,20 +53,21 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: 'My Site',
+        title: 'DIDs',
         logo: {
           alt: 'My Site Logo',
```

Comment:
Assuming it's the Ceramic logo used here
```suggestion
          alt: 'Ceramic logo',
```